_episodes/08-putting-it-all-together.md

```diff
@@ -7,8 +7,11 @@ questions:
     - "Why should I use Python to create plots?"
 objectives:
     - "Import the pyplot toolbox to create figures in Python."
+    - "Use matplotlib to make adjustments to Pandas or plotnine objects."
 keypoints:
-    - "FIXME"
+    - "Matplotlib is the engine behind plotnine and Pandas plots."
+    - "The object-oriented framework of matplotlib makes detailed customization possible."
```

Comment:
I'm not sure how I feel about this keypoint. What would attendees get out of it? Keypoints should be "to the point", that is, related to actions attendees learn during the lesson.